cinemaAppFE/src/app/comment/comment.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, OnInit } from '@angular/core';
+import { HttpRequest } from '@angular/common/http';
+declare var $: any;
+@Component({
+  selector: 'app-comment',
+  templateUrl: './comment.component.html',
+  styleUrls: ['./comment.component.css']
+})
+export class CommentComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+
+}
+function render(data){
+  var html="<div class='commentBox'><div class='leftPanelImg'></div><div class='rightPanel'><span>"+data.name+"</span> <div class='date'>"+data.date+"</div><p>"+data.body+"</p></div><div class='clear'></div></div>";
+  
+  $('#container').append(html);
+}
+
+$(document).ready(function(){
```

Comment:
the code in this fnction should be in the onInit function

onInit is called after the component has loaded (no need ot wait for whole document) often it is the same time as document.ready